README.md

```diff
@@ -165,6 +167,12 @@ optional arguments:
   -m DIR, --merge-into DIR
                         the directory to put the final merged kcov result into
                         (default `target/kcov`)
+  --exclude-pattern PATTERN
+                        pass additional exclusionary patterns to kcov
```

Comment:
Could you change these to describe a little bit more about what the patterns are/what they exclude?